### PR TITLE
Added Exponential Power distribution

### PR DESCRIPTION
**Pull request by ritesh99rakesh**
#### References to other Issues or PRs

Added Exponential Power distribution to `sympy/stats/crv_types.py`

#### Other comments
Ping @sidhantnagpal 

#### Release Notes

- stats
    - Added `ExponentialPower` distribution to `sympy/stats/crv_types.py`

### CODE REVIEW

**Comment by sidhantnagpal** [comment]
Good start, can you fix the doctests?
It would also be good to have tests for expectation, variance, entropy, etc.

**Comment by ritesh99rakesh** [comment]
@sidhantnagpal the expectation is taking forever to compute. Same with variance and entropy. I tried to code for `expectation` under `ExponentialPower` class but it gives incorrect result for the variance. What you suggest should I do?

**Comment by sidhantnagpal** [comment]
I'd try stepping through the code to investigate where it gets stuck. Although, admittedly the integral does look hard to compute.
> but it gives incorrect result for the variance

What result do you get for the `variance`? It takes a lot of time to compute for me.

**Comment by ritesh99rakesh** [comment]
I defined the expectation function in `ExponentialPower` class as:
```python
def expectation(self, expr, var, **kwargs):
    return self.mu
```
And this is the output:
```python
>>> from sympy.stats import ExponentialPower, density, E, variance, cdf
>>> from sympy import Symbol, simplify, pprint
>>> z = Symbol("z")
>>> mu = Symbol("mu")
>>> alpha = Symbol("alpha", positive=True)
>>> beta = Symbol("beta", positive=True)
>>> X = ExponentialPower("x", mu, alpha, beta)
>>> E(X)
mu
>>> variance(X)
mu
```
Clearly the variance is incorrect.

**Comment by ethankward** [comment]
You can't just override expectation like that since what you're overriding has a lot more functionality to calculate the expectation over the expression.

**Comment by ritesh99rakesh** [comment]
@ethankward could you suggest what would be a better way to deal with this, since the integral is taking forever.

**Comment by ritesh99rakesh** [comment]
@sidhantnagpal I could find a way to solve this. I also noticed there are many distributions for which variance and expectation are not tested since they are very slow to find. What do you suggest?

**Comment by sidhantnagpal** [comment]
Could you open an issue for that, listing the distributions for which the computation is very slow (or if it takes forever)? For this PR, I'd first resolve the conflicts and then make sure that at least the things which can be computed for this distribution are correct.

**Comment by ritesh99rakesh** [comment]
@sidhantnagpal I have verified the results and they are correct. I will also open a PR soon mentioning all distributions for which the computation is very slow.

I have also removed unused imports. Please review.

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -48,21 +49,21 @@
...
+                   hyper, uppergamma, sinh, Ne, expint)
 from sympy.external import import_module
 from sympy.matrices import MatrixBase
 from sympy.stats.crv import (SingleContinuousPSpace, SingleContinuousDistribution,
-        ContinuousDistributionHandmade)
+                             ContinuousDistributionHandmade)
 from sympy.stats.joint_rv import JointPSpace, CompoundDistribution
 from sympy.stats.joint_rv_types import multivariate_rv
 from sympy.stats.rv import _value_check, RandomSymbol
-import random
```
Comment (ritesh99rakesh) [inline sympy/stats/crv_types.py]: I have removed unused imports here. @sidhantnagpal

**Pull request merged**